### ISSUE TITLE
Missing support for the Arctis Nova 7 Wireless

### ISSUE THREAD

**Issue by cooldisorganizedmodern** [mtadin/steelseries-wireless-battery-tray#30]
Hello, As I promised, I've got my SteelSeries Arctis Nova 7 Wireless (PC) and your program doesn't support this device. I know you need to gather some information from me in purpose to support this headset so please let me know what I need to send you so this headset will be supported.

Thank You!

**Comment by mtadin** [comment]
Hey, have you tried the newer headsets beta in the releases tab: https://github.com/mtadin/steelseries-wireless-battery-tray/releases? Unfortunately I am still looking for a way to have both newer and older headsets work with the program, hopefully I will have time to dig into this soon

**Comment by cooldisorganizedmodern** [comment]
Yes. I have tried the 'v2.0.1-newerHeadsetsBeta' version and my headset doesn't show up

**Comment by mtadin** [comment]
hi, can you try this latest release, it should work now: https://github.com/mtadin/steelseries-wireless-battery-tray/releases/tag/v3.0.0
